docs/en/install.md

```diff
@@ -10,8 +10,9 @@ The compatible MMClassification and MMCV versions are as below. Please install t
 
 | MMClassification version |     MMCV version      |
 |:------------------------:|:---------------------:|
-| dev                      | mmcv>=1.3.16, <=1.5.0 |
-| 0.19.0 (master)          | mmcv>=1.3.16, <=1.5.0 |
+| dev                      | mmcv>=1.4.4, <=1.5.0  |
+| 0.20.0 (master)          | mmcv>=1.3.16, <=1.5.0 |
```

Comment:
mmcv>=1.4.2

Reply:
we can consider updating the `1.5.0` to `1.6.0` now.

Reply:
CPU training/inference is supported after 1.4.4, So we will change the dependency to 1.4.4 next version.
As for the max version, keep it in this version. May update after discussion.

Reply:
get it.